### ISSUE TITLE
jsx-alignment should allow empty tag.

### ISSUE THREAD

**Issue by zheeeng** [palantir/tslint-react#136]
In React 16, we have Fragments usage. like below:

```jsx
render() {
  return (
    <>
      <ChildA />
      <ChildB />
      <ChildC />
    </>
  );
}
```

jsx-alignment rule will report `[tslint] Tag closing must be on its own line and aligned with opening of tag (jsx-alignment)` at present.

**Comment by zheeeng** [comment]
Oh, I made a mistake. I thought that React support this [usage](https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html), but not. I'm sorry for closing this issue.

**Comment by adidahiya** [comment]
Closing due to old age, please comment if it's still an issue
